lib/extract_messages.dart

```diff
@@ -82,7 +82,19 @@ class MessageExtraction {
   Map<String, MainMessage> parseFile(File file, [bool transformer = false]) {
     // Optimization to avoid parsing files we're sure don't contain any messages.
```

Comment:
This comment applies to the contains("Intl") check below, which is now no longer in this method. It should be moved to the code it describes.